app/build.gradle

```diff
@@ -143,8 +143,9 @@ dependencies {
     implementation "androidx.recyclerview:recyclerview:1.0.0"
     implementation "androidx.coordinatorlayout:coordinatorlayout:1.1.0"
     implementation "androidx.media:media:1.1.0"
+    implementation "androidx.viewpager2:viewpager2:1.0.0"
     implementation "android.arch.work:work-runtime:$workManagerVersion"
-    implementation "com.google.android.material:material:1.0.0"
+    implementation "com.google.android.material:material:1.1.0"
```

Comment:
Had to upgrade material version to use TabLayoutMediator which is added in 1.1.0.